travis/install-sandia-openshmem.sh

```diff
@@ -18,17 +18,14 @@ if [ ! -d "$SHMEM_ROOT" ]; then
     #git clone --depth 10 https://github.com/Sandia-OpenSHMEM/SOS.git sandia-shmem
     #git clone -b v1.3.0-rc1 --depth 10 https://github.com/Sandia-OpenSHMEM/SOS.git sandia-shmem
     #cd sandia-shmem
-    # 1.3 release
-    wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.3.0-rc1.tar.gz
-    tar -xzf v1.3.0-rc1.tar.gz
-    cd SOS-1.3.0-rc1
-    # 1.2 release
-    #wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.2.0.tar.gz
-    #tar -xzf v1.2.0.tar.gz
-    #cd SOS-1.2.0
+    # 1.3.2 release
+    wget https://github.com/Sandia-OpenSHMEM/SOS/archive/v1.3.2.tar.gz
+    tar -xzf v1.3.2.tar.gz
+    cd SOS-1.3.2
     ./autogen.sh
-    # must build in-place (https://github.com/regrant/sandia-shmem/issues/49)
-    ./configure --with-ofi=$TRAVIS_ROOT/libfabric --with-ofi-libdir=$TRAVIS_ROOT/libfabric/lib \
+    mkdir build
+    cd build
+    ../configure --with-ofi=$TRAVIS_ROOT/libfabric --with-ofi-libdir=$TRAVIS_ROOT/libfabric/lib \
```

Comment:
I must be super-stupid, because it is failing with `oshrun: Error, could not find a launcher.  Aborting.`